include/libtorrent/kademlia/get_item.hpp

```diff
@@ -43,7 +43,7 @@ namespace libtorrent { namespace dht
 class get_item : public find_data
 {
 public:
-	typedef boost::function<bool(item&, bool)> data_callback;
+	typedef boost::function<void(item&, bool)> data_callback;
```

Comment:
is nodes_callback called when we receive an authoritative response? and it triggers the put?
for immutable items, what's the distinction between data_callback and nodes_callback?